prover/src/common/prover/chunk.rs

```diff
@@ -1,15 +1,37 @@
 use super::Prover;
 use crate::{
-    config::{LAYER1_DEGREE, LAYER2_DEGREE},
-    utils::gen_rng,
+    config::{asset_file_path, LayerId, LAYER1_DEGREE, LAYER2_DEGREE},
+    utils::{chunk_trace_to_witness_block, gen_rng, get_block_trace_from_file, read_env_var},
 };
 use aggregator::extract_proof_and_instances_with_pairing_check;
 use anyhow::{anyhow, Result};
 use halo2_proofs::halo2curves::bn256::Fr;
+use once_cell::sync::Lazy;
 use snark_verifier_sdk::Snark;
+use std::path::Path;
 use zkevm_circuits::evm_circuit::witness::Block;
 
+static SIMPLE_TRACE_FILENAME: Lazy<String> =
+    Lazy::new(|| read_env_var("SIMPLE_TRACE_FILENAME", "simple_trace.json".to_string()));
+
 impl Prover {
+    pub fn gen_chunk_pk(&mut self, output_dir: Option<&str>) -> Result<()> {
```

Comment:
is this function still relevant? or just keep it here in case for future use?

Reply:
Delete this function in commit [66f6d56](https://github.com/scroll-tech/scroll-prover/pull/245/commits/66f6d561cdf33d512c701faf481a478a0ff81a90).